scripts/deployment20/test-configs/test-aave-v3-mainnet.conf.json

```diff
@@ -96,7 +96,8 @@
         "history": [],
         "constructorArgs": [
           "address:ServiceRegistry",
-          "0x6B175474E89094C44Da98b954EedeAC495271d0F"
+          "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+          "0x5a15566417e6C1c9546523066500bDDBc53F88C7"
```

Comment:
This won't be needed because the constructor is wrong ( there is a param that's not used at all )